core/src/main/scala/org/genivi/sota/core/db/UpdateSpecs.scala

```diff
@@ -228,6 +228,18 @@ object UpdateSpecs {
     createHistoriesIO.andThen(dbIO).transactionally
   }
 
+  def cancelAllUpdatesByRequest(updateRequest: Uuid)
+                               (implicit ec: ExecutionContext): DBIO[Int] =
+    cancelAllUpdatesBy(us => us.status === UpdateStatus.Pending && us.requestId === updateRequest.toJava,
+                       pkg => true)
```

Comment:
argh we should really be using proper types for all these uuids. (not in this pr though)